### PR TITLE
Create Iron Man Jarvis AI Desktop Voice Assistant

### PR DESCRIPTION
**Pull request by kunalkaushik987**

### CODE REVIEW

File: Iron Man Jarvis AI Desktop Voice Assistant [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            songs = os.listdir(music_dir)
+            print(songs)    
+            os.startfile(os.path.join(music_dir, songs[0]))
+
+        elif 'the time' in query:
+            strTime = datetime.datetime.now().strftime("%H:%M:%S")    
+            speak(f"Sir, the time is {strTime}")
+
+        elif 'open code' in query:
+            codePath = "C:\\Users\\Haris\\AppData\\Local\\Programs\\Microsoft VS Code\\Code.exe"
```
Comment (robhaswell) [inline Iron Man Jarvis AI Desktop Voice Assistant:97]: This hardcoded path is specific to your installation.

File: Iron Man Jarvis AI Desktop Voice Assistant [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+        elif 'open google' in query:
+            webbrowser.open("google.com")
+
+        elif 'open stackoverflow' in query:
+            webbrowser.open("stackoverflow.com")   
+
+
+        elif 'play music' in query:
+            music_dir = 'D:\\Non Critical\\songs\\Favorite Songs2'
```
Comment (robhaswell) [inline Iron Man Jarvis AI Desktop Voice Assistant:87]: This hardcoded path is specific to your installation.

**Comment by robhaswell** [comment]
Can you explain why this should be part of the Python image for Docker?

**Comment by ulgens** [comment]
What kind of spam is this?

**Comment by tianon** [comment]
Likely hacktoberfest-inspired; we've been getting a lot (check closed PRs with the invalid label). 😔

**Pull request closed without merging**